tests/lib/tools/os.query

```diff
@@ -88,7 +88,7 @@ is_opensuse() {
 }
 
 is_opensuse_tumbleweed() {
-    grep -qFx 'ID="opensuse-tumbleweed"'
+    grep -qFx 'ID="opensuse-tumbleweed"' /etc/os-release
```

Comment:
This looks fine but I don't is_opensuse_tumbleweed uses anywhere, so I presume it's the other change that fixes problems? To be clear, of course it's fine to land it in this PR.

Reply:
Hi, thanks for the review, it is used here https://github.com/snapcore/snapd/blob/master/tests/lib/dirs.sh#L20